pybamm/models/submodels/interface/sei/base_sei.py

```diff
@@ -73,24 +75,26 @@ def _get_standard_thickness_variables(self, L_inner, L_outer):
             The variables which can be derived from the SEI thicknesses.
         """
         domain, Domain = self.domain_Domain
-        variables = {
-            f"{Domain} inner {self.reaction_name}thickness [m]": L_inner,
-            f"{Domain} outer {self.reaction_name}thickness [m]": L_outer,
-        }
+        L_sei = L_outer
+        variables = {f"{Domain} outer {self.reaction_name}thickness [m]": L_outer}
+
```

Comment:
I wouldn't add outer thickness in the single-layer case, only total thickness

Reply:
or just call it "SEI thickness" (not outer or total)

Reply:
Right, we could rename it to just `SEI thickness`. 
This renaming also apply to the other parameters in the single-layer case, right?
Ex: 
```python
f"X-averaged {domain} electrode outer {self.reaction_name}interfacial current density [A.m-2]"
```
would become:
```python
f"X-averaged {domain} electrode {self.reaction_name}interfacial current density [A.m-2]"
```